src/css/profile/mobile/common/gridview.less

```diff
@@ -164,6 +164,18 @@ tau-gridview {
 		}
 	}
 
+	&.ui-gridview-image-icon .ui-gridview-item img,
```

Comment:
If the girdview image is mixed with general image and icon 
and I don't know which image will be on the grid , how about it?
I think in that case I can't use the "ui-gridview-image-icon".